reorder_python_imports/main.py

```diff
@@ -247,12 +254,30 @@ def apply_import_sorting(partitions, separate_relative=False, **sort_kwargs):
     new_imports = []
     relative_imports = []
 
+    def _is_require_extra_new_line(last_import_obj, import_obj):
```

Comment:
hmmm, to me it felt like the only time you would insert an extra newline is when the type of the imports switches from import imports to from imports, shrugs! maybe `_requires_extra_new_line` is a better name then if you want to keep the original intent